operations/research-and-development/README.md

```diff
@@ -1,20 +1,49 @@
 # Research & Development
 
-R&D includes engineering, QA, product management, and design \(UX, UI\). 
+R&D includes engineering, QA, product management, documentation, analytics, community, and design \(UX, UI\).
 
-## R&D Key Info
+### R&D Key Info
 
-* [Development & Release Process](https://handbook.mattermost.com/operations/research-and-development/product/release-process/release-overview): How open source software is developed
-* [Platform Contribution Process](https://handbook.mattermost.com/contributors/contributors/community-playbook): How contributors can get involved
-* [Analytics](https://community.mattermost.com/private-core/channels/analytics-2): Analytics playbook and data wallows
-* [WIP: Feature Idea Flow Chart](https://docs.google.com/drawings/d/1D6KiN31mhNr1A0DGGHOuu6hvS0gpDTFuftnh2yyniNQ/edit): Work-in-progress
 
 **Product Strategy**
```

Comment:
Let's remove this section for now.  We will want to update it with more current information.  I will work on that separately with Corey.